tutorials/plot_forward.py

```diff
@@ -86,15 +86,26 @@
                        surfaces='head-dense')
 
 ###############################################################################
+# .. _plot_forward_source_space:
+#
 # Compute Source Space
 # --------------------
 #
-# The source space defines the position of the candidate source locations.
-# The following code compute such a cortical source space with
-# an OCT-6 resolution.
+# The source space defines the position and orientation of the candidate source
+# locations. There are two types of source spaces:
+# - **source-based** source space when the candidates are confined to a surface.
+# - **volumetric or discrete** source space when the candidates are discrete,
+#   arbitrarily located source points bounded by the surface.
+#
+# **Source-based** source space is computed using
+# :func:`mne.setup_source_space`, while **volumetric** source space is computed
+# using :func:`mne.setup_volume_source_space`.
 #
+# The rest of this tutorial uses a source-based source space with
+# an OCT-6 resolution.
 # See :ref:`setting_up_source_space` for details on source space definition
```

Comment:
Do we have an example comparing `source-based` and `volumetric` source spaces? I didn't find it.

Reply:
Not explicitly. There are many examples of surface source estimation, and fewer examples of volume source estimation. Some of these operate on the same datasets.